pkg/controller/core/localqueue_controller.go

```diff
@@ -183,6 +183,10 @@ func (r *LocalQueueReconciler) Update(e event.UpdateEvent) bool {
 	log := r.log.WithValues("localQueue", klog.KObj(newLq))
 	log.V(2).Info("Queue update event")
 
+	if features.Enabled(features.LocalQueueMetrics) {
+		updateLocalQueueResourceMetrics(newLq)
+	}
```

Comment:
This is a good catch. Thanks!

I think we also need to cherry pick it somehow. @mimowo WDYT, is it ok to keep it here? Or should we create prepare PR just for this fix?